lib/scale/encode.go

```diff
@@ -103,6 +103,10 @@ func (se *Encoder) EncodeCustom(in interface{}) (int, error) {
 	if ok {
 		res := reflect.ValueOf(in).MethodByName("Encode").Call([]reflect.Value{})
 		val := res[0].Interface()
+		if len(res) < 2 {
```

Comment:
👀

Reply:
yeah had to add this since there was a type that had `Encode() []byte`.... your interface cleanup should fix this